bin/sys-tuner.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Solana System Tuner
+After=network.target
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+User=sol
+StartLimitIntervalSec=0
+LogRateLimitIntervalSec=0
+ExecStart=/home/sol/bin/sys-tuner.sh
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I think we want a `Before=sol` directive, to ensure sys-tuner starts before the validator process, as the validator process only pokes sys-tuner at startup